variables.tf

```diff
@@ -7,3 +7,9 @@ variable "dataset_id" {
   type        = string
   description = "The dataset that Fullstory will use to sync data."
 }
+
+variable "service_account_name" {
+  type = string
+  description = "The name of the service account to use with Fullstory"
+  default = "fullstory-bigquery-svc"
```

Comment:
@camphillips22 do you think naming this the same name as the module is helpful for tracking down what may have created it? I think `svc` may be a little redundant since the consumer should know that it's a service account by virtue of the entity type.

Reply:
Fair enough! I'll update it to be `fullstory-bigquery-setup` then.